roles/zuul/files/jobs/bonnyci.yaml

```diff
@@ -10,7 +10,10 @@
               copy-after-failure: true
 
 - job-template:
-    name: 'bonnyci-run-{pipeline}'
+    node: 'ubuntu-xenial'
```

Comment:
Yeah, this is just the default value.  Passing a node in below will overwrite it.  Verified this locally by referencing it in the executor and inspecting the generated job.